port/common/omrgetsysname.h

```diff
@@ -25,6 +25,7 @@
 
 #include "omrcomp.h"
 
+struct OMRPortLibrary;
```

Comment:
A blank line after this one would be welcome.

Reply:
Done